src/routes/users.js

```diff
@@ -5,14 +5,14 @@ const {
 } = require('../controller/usersController');
 
 const express = require('express');
+const { usersService } = require('../services/users');
 
 // #region usersRouter
 const usersRouter = express.Router();
 
 function users(app) {
   app.use('/users', usersRouter);
-  usersController.setInitialUsers();
-  usersRouter.use(express.json());
+  usersService.setInitialUsers();
```

Comment:
```suggestion
```

Reply:
> Small fixes and you will be good to go.

look please on test logic! it rerun function not a Server! it's very important! I cann't transfer objects created in function between files and I need to connect Service and Controller logic! I can avoid reset of this in class object's but tests will not work!! 
once more 1) look on test logic! 2) It's not my decision to rewrite every this of data, I did it only to pass the test logic! 3) test logic must simulate RESTART of the server, not recall of the function! 4) I and many of student's, we told this to mentors, also we wrote this in qna chanel! Mentors told to skip test logic and tests because they not simulate server close/open.